### ISSUE TITLE
when I added tow spotlight in the scene the didn't work

### ISSUE THREAD

**Issue by mhreez** [armory3d/armory#1429]
Thank you for contributing to Armory!

**Description**
Issue description.
I have added a spotlight to the torch . it is working no problem
when I added another spot light to Lamppost I discovered that all the spotlight did not work
I've tried this on more than one device Still did not work
and some time when I add one of spotlight didn't work Just one 

**To Reproduce**
Steps to reproduce the behavior.
just add tow spotight
[test 03 flash light.zip](https://github.com/armory3d/armory/files/3694910/test.03.flash.light.zip)

**Expected behavior**
What you expected to happen.

**System**
Blender:
Armory:
OS:
Graphics card:

**Test File**
*(drag & drop the zipped .blend file here)*

**Comment by Sandy10000** [comment]
The zip file is broken and could not be confirmed. Is this a problem different from the following problems?
https://github.com/armory3d/armory/issues/1289

**Comment by mhreez** [comment]
thank you sandy
it is working for me 
any way you cane get the file from here ( winrar file )
https://www.mediafire.com/file/bx43srjuja7req2/test_03_flash_light.rar/file

**Comment by mhreez** [comment]
> The zip file is broken and could not be confirmed. Is this a problem different from the following problems?
> #1289

https://www.mediafire.com/file/bx43srjuja7req2/test_03_flash_light.rar/file

**Comment by Sandy10000** [comment]
Is the content of the bug you are trying to report as follows?Please feel free to point out if they are different.

1.  Adding a spotlight does not reflect. However···
![1](https://user-images.githubusercontent.com/48373918/67098747-b1941980-f1f7-11e9-9ee1-ed516aaadf2b.jpg)
1.  When press the "f" key to turn on the first spotlight, the added spotlight also turns on. Repeatedly pressing the “f” key to turn on / off the first spotlight will also cause the added spotlight to turn on / off
![2](https://user-images.githubusercontent.com/48373918/67098943-08015800-f1f8-11e9-83b1-ba3d6e97ef8b.jpg)
1.  If add a third spotlight, the second spotlight will always be on (correct operation) and the third spotlight will be linked to the first spotlight on / off.
![3](https://user-images.githubusercontent.com/48373918/67099700-61b65200-f1f9-11e9-981a-817b96f85c59.jpg)
![4](https://user-images.githubusercontent.com/48373918/67099706-64b14280-f1f9-11e9-8252-8276feb62139.jpg)  
In other words, the last added spotlight is turned on / off in conjunction with the spotlight that is controlled on / off.

**Comment by Sandy10000** [comment]
Although it is not a complete solution, the problem can be avoided by setting the power value and strength value when the spotlight to be controlled on / off is off to a small value other than 0 (such as 0.0001).

**Comment by mhreez** [comment]
thank you sandy

**Comment by knowledgenude** [comment]
Follow https://github.com/armory3d/armory/issues/1772#issue-660023398
